test/validators.test.ts

```diff
@@ -0,0 +1,44 @@
+import { failure, isFailure, isSuccess, success } from "../src/Result";
+import { validators } from "../src/validators";
+
+describe("validators", () => {
+  describe("inSequence", () => {
+    test("it should success validation", async () => {
```

Comment:
maybe "succeed"? or just simply (since it's the first basic test for this validator):
```suggestion
    test("it should work", async () => {
```

Reply:
failure test is also a "it should work" basic scenario isn't it? 😄 I'll go with "succeed"